src/groups/mqb/mqbcfg/mqbcfg.xsd

```diff
@@ -303,6 +305,120 @@
+        Top level type for the broker's authentication configurations.
+
+        authenticators...........:
+            Configuration entries for authenticator plugins (built-in or external).
+            Each entry defines settings for a specific plugin. All plugins must have
+            unique authentication mechanisms.
+        anonymousCredential.:
+            Controls anonymous authentication behavior. When specified, the broker
+            uses the provided credential with a matching plugin from `authenticators`.
+            When omitted, the broker defaults to AnonAuthenticator and always passes for
+            anonymous authentication.
+      </documentation>
+    </annotation>
+    <sequence>
+      <element name='authenticators'      type='tns:AuthenticatorPluginConfig' minOccurs="0" maxOccurs='unbounded'/>
+      <element name='anonymousCredential' type='tns:AnonymousCredential' minOccurs='0'/>
+     </sequence>
+  </complexType>
+
+  <complexType name='AnonymousCredential'>
+    <annotation>
+      <documentation>
+        Type representing the anonymous credential configuration.
+
+        disallow...:
+            If set, the anonymous credential is not allowed. Authentication
+            is required and clients which cannot or do not authenticate will be
+            rejected.
+        credential.:
+            If set, the credential is used for anonymous authentication in case the client
+            does not support authentication or has not been configured to authenticate.
+      </documentation>
+    </annotation>
+    <choice>
+      <element name='disallow'   type='tns:Disallow'/>
+      <element name='credential' type='tns:Credential'/>
+    </choice>
+  </complexType>
+
+  <complexType name="Credential">
+    <annotation>
+      <documentation>
+        Type representing a credential used for authentication.
+
+        This type is used to represent a credential that can be used for
+        authentication. It contains an authentication mechanism and an identity.
+      </documentation>
+    </annotation>
+    <sequence>
+      <element name='mechanism' type='string'/>
+      <element name='identity' type='string'/>
+    </sequence>
+  </complexType>
+
+  <complexType name="Disallow">
+    <annotation>
+      <documentation>
+        Type representing the disallow anonymous credential configuration.
+
+        This type is used to indicate that anonymous authentication is not
+        allowed on the broker. If this is set, the broker will not use the
+        anonymous authenticator plugin. Authentication is required and clients
+        which cannot or do not authenticate will be rejected.
+      </documentation>
+    </annotation>
+    <sequence></sequence>
+  </complexType>
+
+  <complexType name="AuthenticatorPluginConfig">
+    <annotation>
+      <documentation>
+        The configuration for an authenticator plugin.
+
+        name....:
+            The name of the authenticator plugin.
+        configs.:
```

Comment:
I still think that `settings` is a better name here. Currently:
1. There is a general broker config
2. There is a plugin config
3. Each plugin config might have multiple configs (here)